packages/api/package.json

```diff
@@ -40,12 +40,12 @@
   },
   "devDependencies": {
     "ts-expect": "^1.3.0",
-    "typescript": "^5.2.2"
+    "typescript": "^5.4.2"
   },
   "publishConfig": {
     "access": "public"
   },
-  "keywords": [ ],
+  "keywords": [],
```

Comment:
Reminder that we still don't have autoformatting enabled for package.json because of autorelease